liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -164,9 +174,19 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
 	// exception header line
-	errorColored() << _msg.severity;
+	errorColored() << _msg.severity_error;
+    warningColored() << _msg.severity_warning;
+    infoColored() << _msg.severity_info;
```

Comment:
What are `severity_error`, `severity_warning` and `severity_info`? `SourceReferenceExtractor::Message` does not have these fields.

Reply:
Fixed 👍